### ISSUE TITLE
Didn't find a way to create PaymentIntent from from token

### ISSUE THREAD

**Issue by idrish-sorathiya** [stripe/stripe-node#1310]
I am getting token from Apple and Google Pay. From that I wanted to created PaymentIntent but didnt find a way to do it in Node library version 8.194.0.  `PaymentIntentCreateParams` has card but card has no parameter for token. 

Looking for something like,
```
curl https://api.stripe.com/v1/payment_intents \
  -u sk_test_xxx: \
  -d amount=2000 \
  -d currency=usd \
  -d "payment_method_types[]"=card \
  -d confirm=true \
  -d "payment_method_data[type]"="card" \
  -d "payment_method_data[card][token]"="tok_visa"
```
Please let me know if there is other way or I am doing something wrong into this.

**Comment by remi-stripe** [comment]
@idrish-sorathiya the code you shared will work fine but we don't have types for this. This is a flow using the legacy Token API and something we mostly discourage using so there are no types in stripe-node for it. We comment creating the PaymentIntent server-side and then confirming it client-side either in your browser in JS or in your mobile app instead.

```
const paymentIntent = await stripe.paymentIntents.create({
  amount: 2000,
  currency: 'usd',
  payment_method_types: ['card'],
  confirm: true,
  payment_method_data: {
    type: 'card',
    card: {
      token: 'tok_visa',
    },
  },
});
```

**Comment by idrish-sorathiya** [comment]
Sorry but ignoring type and pass this body so, received error: **"Received unknown parameter: payment_method_options[type]**

Maybe complaining about "type: card"

I looked at doc but didn't  find any valid value for it.

**Comment by idrish-sorathiya** [comment]
I was passing wrong parameter so, I corrected it but still getting error - **Error: Missing required param: type.**

**Comment by remi-stripe** [comment]
@idrish-sorathiya the code I shared above works, I've tested it before sharing it with you. If your code isn't working it' likely because you aren't passing the `type` parameter even though you think you might be.
Try hardcoding exact values like I did and it will work.

There's no bug in the library here, you're asking for integration help so you would talk to our support team directly in that case: https://support.stripe.com/contact
